### PR TITLE
Readme update [no JIRA]

### PR DESCRIPTION
**Pull request by aednichols**
- Updated language for things that are no longer new, or don't seem to be happening
- Promote JIRA with a screenshot so users are confident there really is a promised land after they create an account
- Reduced exclamation

IntelliJ preview:

Screen Shot 2020-12-08 at 5 23 11 PM

### CODE REVIEW

**Review by grsterin — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -3,20 +3,26 @@
 
 ## Welcome to Cromwell
 
-Cromwell is a Workflow Management System geared towards scientific workflows. Cromwell is open sourced under the [BSD 3-Clause license](LICENSE.txt).
+Cromwell is a Workflow Management System for bioinformatics. Cromwell is open source under the [BSD 3-Clause license](LICENSE.txt).
```
Comment (rsasch) [inline README.md:6]: nit
```suggestion
Cromwell is an open source (under the [BSD 3-Clause license](LICENSE.txt)) Workflow Management System for bioinformatics.
```

File: README.md [hunk 2]
```diff
@@ -3,20 +3,26 @@
...
 
-The Cromwell documentation has a new home, [click here to check it out](https://cromwell.readthedocs.io/en/stable)!
+The Cromwell documentation has a dedicated site, [click here to check it out](https://cromwell.readthedocs.io/en/stable).
```
Comment (rsasch) [inline README.md:8]: nit for accessibility
```suggestion
The [Cromwell documentation](https://cromwell.readthedocs.io/en/stable) has a dedicated site.
```

**Review by rsasch — APPROVED** [review]
small nits for clarity/accessibility, otherwise 👍

**Comment by aednichols** [comment]
@rsasch I did not implement your suggestions literally but I hope I captured the spirit

**Review by rsasch — APPROVED** [review]
Looks good — you indeed captured the spirit of my changes 😄

File: README.md [hunk 3]
```diff
@@ -3,20 +3,26 @@
...
+<!--
+AEN external issue filing tested 2020-12-08 with `oednichols@gmail.com` / `https://broadworkbench.atlassian.net/browse/CROM-6681`
+-->
 
-As of May 2019, we are in the process of migrating all issues from Github to JIRA. At a later date to be announced, submitting new Github issues will be disabled.
+Need to file an issue? Head over to [our JIRA](https://broadworkbench.atlassian.net/jira/software/c/projects/CROM/issues). You must create a free profile to view or create.
+
+![Cromwell JIRA](docs/img/cromwell_jira.png)
+
+Issues in Github remain available for discussion among community members but are not actively monitored by the development team.
```
Comment (cjllanwarne) [inline README.md:26]: There's probably an `ISSUES.MD` template or something we could add to make this even clearer... Or just turn off issues in the repo

**Pull request merged**